octobot/task_manager.py

```diff
@@ -91,21 +92,33 @@ async def stop_timeout(timeout):
             stop_coroutines.append(self.octobot.community_handler.stop_task())
 
         async def _await_timeouted_gather(tasks):
-            # await this gather to be sure to complete the each stop call or timeout
+            # await this gather to be sure to complete each stop call or timeout
             try:
                 await asyncio.gather(*tasks)
             except asyncio.exceptions.TimeoutError:
                 self.logger.warning(f"Timeout while stopping tasks, forcing stop.")
+                raise
```

Comment:
Why should we raise here?

Reply:
To trigger this block only on error
![image](https://user-images.githubusercontent.com/9078616/193409679-9c2f036d-2eca-4bb8-a4f4-8849062e7a57.png)

Reply:
👍